src/DialogManager.php

```diff
@@ -40,6 +40,17 @@ public function activate(Dialog $dialog): void
         $this->storeDialogState($dialog);
     }
 
+    /**
+     * @api Deactivate existing Dialog.
+     */
+    public function deactivate(Update $update): void
+    {
+        $dialog = $this->getDialogInstance($update);
+        if(isset($dialog)){
```

Comment:
```suggestion
        if ($dialog instanceof Dialog) {
```
according best practices, the only one valid usage of isset is to check whether a variable exist in the context.